cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -596,6 +604,14 @@
   </property>
 
   <property>
+    <name>dataset.service.bind.port</name>
+    <value>0</value>
+    <description>
+      Dataset service bind port
+    </description>
+  </property>
+
+  <property>
```

Comment:
The travis build failed... looks like we also need a default for dataset.executor.bind.port.